internal/service/cluster/data_source_cluster.go

```diff
@@ -25,7 +27,7 @@ func DataSource() *schema.Resource {
 				Type:     schema.TypeString,
 				Required: true,
 			},
-			"advanced_configuration": advancedcluster.ClusterAdvancedConfigurationSchemaComputed(),
+			"advanced_configuration": clusterAdvancedConfigurationSchemaComputed(),
```

Comment:
i don't see the method being removed from advancedcluster package, are we duplicating the functions?

Reply:
yes, advanced cluster will be refactored to framework in follow-up PRs, updated description